.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,160 @@
+### Contributing to ZFS on Linux
+
+<p align="center"><img src="http://zfsonlinux.org/images/zfs-linux.png"/></p>
+
+*First of all, thank you for taking the time to contribute!*
+
+The following is a set of guidelines for contributing to ZFS on Linux. 
+Using them you can help us to make a better work.
```

Comment:
Perhaps something like `By using the following guidelines, you can help us make ZFS on Linux even better.`?